src/core/modes/walk-mode.ts

```diff
@@ -125,14 +143,19 @@ export class WalkMode implements Mode {
         this.dispose();
         return;
       }
+      const gaugeColorName = vscode.workspace.getConfiguration('junbae-mode').get('gaugeColor');
```

Comment:
I think we need unify word. We use two different word, `gauge` and `timer`.
If you prefer `gauge`, please modify all related word.

Reply:
It doesn't matter anything

Reply:
ok. use timer plz